### PR TITLE
build: fix minor naming issue in cmake file

### PR DESCRIPTION
**Pull request by mguc**
Fixes the following error when building the `nrf_profiler`:

```
CMake Error at /opt/ncs/zephyr/cmake/modules/extensions.cmake:418 (add_library):
  Cannot find source file:
    /opt/ncs/nrf/subsys/nrf_profiler/nrf_profiler_common_shell.c
Call Stack (most recent call first):
  /opt/ncs/zephyr/CMakeLists.txt:105 (zephyr_library_named)
```

### CODE REVIEW

File: subsys/nrf_profiler/CMakeLists.txt [hunk 1]
```diff
@@ -5,4 +5,4 @@
 #
 
 zephyr_sources_ifdef(CONFIG_NRF_PROFILER_NORDIC profiler_nordic.c)
-zephyr_sources_ifdef(CONFIG_SHELL nrf_profiler_common_shell.c)
+zephyr_sources_ifdef(CONFIG_SHELL profiler_common_shell.c)
```
Comment (pdunaj) [inline subsys/nrf_profiler/CMakeLists.txt:8]: The source file should be renamed instead.

**Review by pdunaj — CHANGES_REQUESTED** [review]

**Comment by pdunaj** [comment]
Hi @mguc , thank you for your contribution. It seems one file was not committed correctly when profiler rename was done. Can you change the fix to adjust the file name instead?

**Review by pdunaj — CHANGES_REQUESTED** [review]
Change is ok but the commits cleanup is neeed.
Please squash two commits into one (`git rebase -i HEAD~2` and mark the second to be squashed). Also re-write first commit's message to match with previous ones. That can be something like:
```
    profiler: Fix source file name.

    Fix profiler source file name according to the recent
    rename.
    
    Signed-off-by: Your Name <your@address.xy>
```

When you commit you can add `-s` (`git commit -s`) for sign-off line to be added for you
according to name/email variables set in git.

**Review by MarekPieta — COMMENTED** [review]
Please use `nrf_profiler` instead of `profiler` as commit message prefix. The subsystem is currently called `nrf_profiler`

**Review by pdunaj — APPROVED** [review]

**Review by MarekPieta — APPROVED** [review]

**Comment by pdunaj** [comment]
Hi @MarekPieta , it would be good to have it in the coming release, could you rebase? Unless @mguc will do it :)

**Comment by MarekPieta** [comment]
Hi @pdunaj
The issue is fixed in https://github.com/nrfconnect/sdk-nrf/pull/9473. (it could go together with profiler shell fix)

**Comment by nordicjm** [comment]
Closing as fixed in #9473

**Pull request closed without merging**